core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1131,14 +1131,43 @@ private[spark] class BlockManager(
   }
 
   /**
+   * Called for pro-active replenishment of blocks lost due to executor failures
+   *
+   * @param blockId blockId being replicate
+   * @param existingReplicas existing block managers that have a replica
+   * @param maxReplicas maximum replicas needed
+   * @return
+   */
+  def replicateBlock(
+    blockId: BlockId,
```

Comment:
Same as Sameer's comment elsewhere in the code, we should fix the indentation here.